platform/broadcom/saibcm-modules/debian/opennsl-modules-4.9.0-8-2-amd64.init

```diff
@@ -64,7 +64,7 @@ force-reload|restart)
     ;;
 
 *)
-    echo "Usage: /etc/init.d/opennsl-modules-4.9.0-8-amd64.init {start|stop}"
+    echo "Usage: /etc/init.d/opennsl-modules-4.9.0-8-2-amd64.init {start|stop}"
```

Comment:
Using "Usage: $0 {start|stop}" would remove the need to change this file every time the kernel is upgraded.

Also is there a reason I'm not seeing as to why the version of the kernel is included in the name of this systemd unit?
Everytime it changes, all dependent services have to change their systemd unit file to match. It seems unnecessary since waiting for the opennsl drivers to be loaded is the same independently of the kernel version.

Reply:
removed the kernel versions